src/app/components/status-indicator/index.html

```diff
@@ -1,31 +1,170 @@
-<sky-demo-page
-  pageTitle="Status Indicator"
-  navTitle="Status indicator"
-  packageName="@skyux/theme"
-  [installOnly]="true"
+<sky-tabbed-demo-page
+  pageTitle="Status indicator"
 >
 
 <sky-demo-page-summary>
-	The status indicator classes provide icons to draw attention to status information that users need to address. Statuses indicate a state or condition at a particular time and fall into four categories: danger, info, success, and warning. For example, a status can indicate that a submission is late, submitted, or awaiting approval.
+	The status indicator classes provide icons to highlight important status information that users need to address.
```

Comment:
The spacing here is off. The tabs are too tight on this text because there is no demo section between the overview paragraph and the tabs. I'm going to add `p` tags around this text because that gives space between the paragraph and the tabs.